### PR TITLE
Completed javascript timers exercise.

### PR DESCRIPTION
**Pull request by vivianevieira**

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var h1 = document.querySelector('h1');
+
+function countdown() {
+  if (h1.textContent === '4') {
```
Comment (codingcodymiller) [inline javascript-timers/interval.js:4]: This is a very manual approach. How can you do this where you don't need a condition for every number? What if we wanted to count down from 1000? That would be horrifying! Consider that you can have variables that change in value!

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var h1 = document.querySelector('h1');
+
+function countdown() {
+  var newH1 = h1.textContent - 1;
+  h1.textContent = newH1;
+  if (h1.textContent === '0') {
+    h1.textContent = '~Earth Beeeelooowww Us~';
+  }
+  if (h1.textContent === '~Earth Beeeelooowww Us~') {
+    clearInterval(interval);
```
Comment (codingcodymiller) [inline javascript-timers/interval.js:10]: Why not just put this into the above conditional?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-timers/interval.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+var h1 = document.querySelector('h1');
+
+function countdown() {
+  var newH1 = h1.textContent - 1;
```
Comment (thebearingedge) [inline javascript-timers/interval.js:4]: Instead of leaving the "truth" in the DOM, use a variable to store the current countdown number.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: javascript-timers/interval.js (hunk 3 above)
Comment (vivianevieira) [reply javascript-timers/interval.js]: Makes sense! Just pushed the new file.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**